Zoom Buttons/ZoomButtons.js

```diff
@@ -81,121 +92,158 @@ tau
+                // mark active button
                 setZoomHighlight( zoom );
 
-
+                // done, close the menu which we opened
                 closeActionMenu();
 
                 // start polling sync when menu is toggled, only open, only if triggered manually
-                $('.tau-board-header__control--actions .tau-actions-btn:not(.tau-checked)').on('click', function() {
-                    // console.log('toggle menu');
+                $(zoom_actionbutton_off_id).on('click', function() {
+
+                    // poll for slider-changes when it was manually opened
                     if( !$(this).hasClass('zoom-open') ) {
-                        // console.log('start poll');
-                        zoomTimer = setTimeout(checkZoomButton, 1000);  // test if slider has changed
-                    } else {
-                        // console.log('no poll');
+
+                        // test if slider has changed
+                        zoom_timer = setTimeout(checkZoomButton, 1000);
                     }
 
                     return true;
                 });
 
-
+                // successfully initialized
                 return true;
+
             } else {
-                // console.log('menu is ok');
-                if ( tries > 0 ) {
-                    setTimeout( function() { waitForSlider( tries -1 ) }, 1000 );  // try again, slider was not ready
+
+                // Did not work, maybe page was not ready
+                // try 10 times and then give up, probably wrong board type (eg. tree or list)
+
+                if ( zoom_tries > 0 ) {
+
+                    // try again, slider was not ready
+                    zoom_wait = setTimeout( waitForSlider, 1000 );
+
                 } else {
-                    // give up , to many tries without menu
+
+                    // give up , too many tries without menu
                     closeActionMenu();
                 }
 
+                // we gave up
                 return false;
             }
         };
 
+        var isActionMenuOpen = function() {
+            return $(zoom_actionbutton_on_id).length !== 0;
+        };
 
+        var openActionMenuOrKeep = function( was_open ){
+            if ( !was_open ) {
+                openActionMenu();
+            }
+        };
 
-        var isActionMenuOpen = function() {
-            return $('.tau-board-header__control--actions .tau-actions-btn.tau-checked').length;
+        var closeActionMenuOrKeep = function( was_open ) {
+            if ( !was_open ) {
+                closeActionMenu();
+            }
         };
 
         var openActionMenu = function() {
-            $('.tau-board-header__control--actions .tau-actions-btn:not(.tau-checked)').addClass('zoom-open'); // mark
-            $('.tau-board-header__control--actions .tau-actions-btn:not(.tau-checked)').click(); // open
+            $(zoom_actionbubble_id).css('opacity',0);               // make menu invisible (but open)
```

Comment:
Suggest to save old opacity here and restore on menu close.
So even if we change menu opacity in the future, mashup continue to work.

Reply:
OK, I can store the old value in temp and restore it

Reply:
The old opacity value is now stored, and restored later